ios_app/barista/CheckinViewController.swift

```diff
@@ -17,17 +17,17 @@ class CheckinViewController: UIViewController {
     @IBOutlet weak var priceLabel: UILabel!
     @IBOutlet weak var checkinPhotoView: UIImageView!
     
-    override func viewWillAppear(animated: Bool) {
+    override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         self.title = checkin.method
         methodLabel.text = checkin.method
         originLabel.text = checkin.origin
         priceLabel!.text = "$ \(checkin.price)"
-        dispatch_async(dispatch_get_global_queue( DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), {
-            dispatch_async(dispatch_get_main_queue(), {
-                let myImage =  UIImage(data: NSData(contentsOfURL: NSURL(string:self.checkin.urlPhoto!)!)!)
+        DispatchQueue.global( qos: DispatchQoS.QoSClass.default).async(execute: {
```

Comment:
Sí, `priority`me lo marcaba como deprecado en iOS 8. Mira: 
* [Modernize libdispatch for Swift 3 naming conventions](https://github.com/apple/swift-evolution/blob/master/proposals/0088-libdispatch-for-swift3.md)
* [Concurrent Programming With GCD in Swift 3](https://youtu.be/zfCZTnEZ6Dw)